mapper/orm/src/main/java/org/hibernate/search/mapper/orm/session/impl/HibernateOrmSearchSession.java

```diff
@@ -213,6 +220,11 @@ public Session toOrmSession() {
 		return sessionImplementor;
 	}
 
+	@Override
+	public SessionImplementor session() {
+		return sessionImplementor;
+	}
+
```

Comment:
we have these publicly exposed ... so I wonder what should we do about them ... (once we start the stateless session work...)